pkg/monitor/datapath_drop.go

```diff
@@ -62,7 +64,6 @@ var sourceFileNames = map[int]string{
 	105: "nodeport.h",
 	106: "lb.h",
 	107: "mcast.h",
-	//end
```

Comment:
Uber nit since we already touch this line:
Why not use `// @@ source files list end` and match against that instead of `}` in the awk script to have the same matching mechanism in both files?